ext/background/background.js

```diff
@@ -140,18 +140,27 @@ chrome.runtime.onMessage.addListener((request, _sender, sendResponse) => {
     case MESSAGE_TYPES.SESSIONS:
       getSessions()
         .then(async (sessions) => {
-          const c = await currentSession();
-          sessions.forEach((s) => {
-            if (s) s.active = s.org.slug == c.org.slug;
-          });
-          sendResponse(sessions);
+          let resp = sessions;
+          if (sessions.length > 1) {
+            try {
+              const c = await currentSession();
+              resp = sessions
+                .filter((s) => s)
+                .map((s) => {
+                  return { ...s, active: s.org.slug == c.org.slug };
+                });
+            } catch (err) {
+              console.log(err);
```

Comment:
Add a code comment here describing why execution continues. It's not obvious to me as a reader.